test/extensions/tracers/zipkin/span_buffer_test.cc

```diff
@@ -176,7 +204,7 @@ TEST(ZipkinSpanBufferTest, SerializeSpan) {
                            R"("traceId":"0000000000000001",)"
                            R"("id":"0000000000000001",)"
                            R"("kind":"CLIENT",)"
-                           R"("timestamp":1566058071601051,)"
+                           R"("timestamp":DEFAULT_TEST_TIMESTAMP,)"
```

Comment:
I might be being dense. I am not sure I understand how this expectation is met given that the DEFAULT_TEST_TIMESTAMP string is not being replaced?

Reply:
Sorry for not being really clear about this. But the `array_string` param of  `wrapAsObject` will replace each `DEFAULT_TEST_TIMESTAMP` placeholder with `DEFAULT_TEST_TIMESTAMP` value above (by calling `withDefaultTimestamp` function), i.e. in this case: `1584324295476870`.

Reply:
Nope, that is my bad, I didn't connect the two 🤦‍♂